mewarpx/examples/capacitive_discharge_copy.py

```diff
@@ -0,0 +1,237 @@
+"""
```

Comment:
If this file is going to be committed/pushed to the remote repo, you should give it a more descriptive name.